nyx_client/nyx_client/circles.py

```diff
@@ -0,0 +1,87 @@
+# Copyright © 2024 IOTIC LABS LTD. info@iotics.com
```

Comment:
Q: Should this be exposed as its own sub-module in the docs? (Currently `RemoteHost` & `Circle` are included at client level due to imports) :thinking:

Reply:
Ah ok - this must be because the module doesn't have a top-level docstring maybe? :thinking: